src/features/profile/services/profile.service.ts

```diff
@@ -0,0 +1,52 @@
+import { BACKEND_V2_URL } from "app/config";
+import { BackendError } from "app/exceptions";
+import axios from "axios";
+import { User } from "core/users/types";
+
+const URL = `${BACKEND_V2_URL}/profiles`;
+
+export async function fetchOneProfile(
+  token: string,
+  profile_id: number
+): Promise<OneProfileResponse> {
+  try {
+    const response = await axios.get(URL + "/" + profile_id, {
```

Comment:
Es mas facil manejarlo asi
```suggestion
    const response = await axios.get(`${URL}/${profile_id}`, {
```

Reply:
true